_rules/button-accessible-name-97a4e1.md

```diff
@@ -21,10 +21,12 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to elements that are [included in the accessibility tree][] and have a [semantic role](#semantic-role) of `button`, except for `input` elements whose `type` attribute is in the [`Image Button` state](<https://html.spec.whatwg.org/multipage/input.html#image-button-state-(type=image)>).
+The rule applies to elements that are [included in the accessibility tree][] and have a [semantic role](#semantic-role) of `button`, including `input` elements whose `type` attribute is in the [`Image Button` state](<https://html.spec.whatwg.org/#image-button-state-(type=image)>).
```

Comment:
```suggestion
The rule applies to elements that are [included in the accessibility tree][] and have a [semantic role](#semantic-role) of `button`.
```
`button` is the implicit role for these elements, I don't think we need the precision.

Reply:
I've added the word "all" to the change. I agree, but wanted to gauge opinion.